src/Turbo/composer.json

```
@@ -0,0 +1,82 @@
+{
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/azjezz/mercure-bundle"
+        }
+    ],
+    "name": "symfony/ux-turbo",
+    "type": "symfony-bundle",
+    "description": "Hotwire Turbo integration for Symfony",
+    "keywords": [
+        "symfony-ux",
+        "turbo",
+        "hotwire",
+        "javascript",
+        "turbo-stream",
+        "mercure"
+    ],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "kevin@dunglas.fr"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/config": "^5.2",
+        "symfony/dependency-injection": "^5.2",
+        "symfony/http-kernel": "^5.2",
+        "symfony/property-access": "^5.2",
+        "symfony/webpack-encore-bundle": "^1.11"
+    },
+    "autoload": {
+        "psr-4": {
+            "Symfony\\UX\\Turbo\\": ""
+        },
+        "exclude-from-classmap": [
+            "/Tests/"
+        ]
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "App\\": "Tests/app/"
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-main": "1.1-dev"
+        },
+        "thanks": {
+            "name": "symfony/ux",
+            "url": "https://github.com/symfony/ux"
+        }
+    },
+    "config": {
+        "sort-packages": true
+    },
+    "require-dev": {
+        "doctrine/doctrine-bundle": "^2.2",
+        "doctrine/orm": "^2.8",
+        "phpstan/phpstan": "^0.12",
+        "symfony/debug-bundle": "^5.2",
+        "symfony/expression-language": "^5.2",
+        "symfony/form": "^5.2",
+        "symfony/framework-bundle": "^5.2",
+        "symfony/mercure-bundle": "dev-feature/token-factory",
```

```suggestion
        "symfony/mercure-bundle": "^0.3",
```

( https://github.com/symfony/mercure-bundle/pull/42/commits/5261c424d626e142def2271fa3179e6d6eb9f43f )